draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1831,7 +1831,9 @@ might block third-party cookies entirely by refusing to send Cookie header field
 process Set-Cookie header fields during third-party requests. They might take a less
 draconian approach by partitioning cookies based on the first-party context,
 sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another.
+and another to the same third party in another. Or they might even allow or set
```

Comment:
> Or they might even allow or set

Why not just "Or they might even allow"?

Reply:
Sigh, because I'm multi-tasking. That was unintentional. 😭